### DISCUSSION TITLE
login to server failed

### DISCUSSION THREAD

**Discussion by xuehy** [natfrp/wiki#153] [使用问题]
今天突然出现用frpc连接一直失败，错误信息：
```frpc[14868]: 2022/08/30 17:44:47 [W] [service.go:128] login to server failed: i/o deadline reached```

反复尝试几次之后会出现登录成功但有其他错误：
```frpc[11559]: 2022/08/30 17:25:00 [W] [control.go:179] [*-aea27a08] [*.next] start error: proxy conflict```
这个错误出现的时候可以确保没有其他frpc进程
请问这是怎么回事呢

**Comment by fakeboboliu** [comment]
#<span/>53 成都电信4 节点可能正在被攻击，目前网络并不稳定

请考虑迁移一下节点
